UsbAccess/UsbAccess.cpp

```diff
@@ -31,7 +32,8 @@ const string WPEFramework::Plugin::UsbAccess::EVT_ON_USB_MOUNT_CHANGED = "onUSBM
 const string WPEFramework::Plugin::UsbAccess::EVT_ON_ARCHIVE_LOGS = "onArchiveLogs";
 const string WPEFramework::Plugin::UsbAccess::REGEX_BIN = "[\\w-]*\\.{0,1}[\\w-]*\\.bin";
 const string WPEFramework::Plugin::UsbAccess::REGEX_FILE =
-				"^[^.].*\\.(png|jpg|jpeg|tiff|tif|bmp|mp4|mov|avi|mp3|wav|m4a|flac|mp4|aac|wma|txt|bin|enc|ts)$";
+				"^[a-zA-Z0-9_-]+\\.(png|jpg|jpeg|tiff|bmp|mp4|mov|avi|mp3|wav|m4a|flac|aac|wma|txt|bin|enc|ts)$";
```

Comment:
The regex is set as open to access all files names after adding Chinese file name support (https://ccp.sys.comcast.net/browse/RDKTV-22205)
Have you verified this modification with Chinese file names on Hisense/Element TVs?